### DISCUSSION TITLE
Numbering of desktops

### DISCUSSION THREAD

**Discussion by HappyMaxxx** [JaKooLit/Hyprland-Dots#493] [Q&A] [answered]
I have a question about how I can change the part at the top where it says desktop numbers, they are written in Japanese, how to change them to normal.

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/issues/492

answered here

but then 
https://github.com/JaKooLit/Hyprland-Dots/wiki/HOW-TO#waybar-costumization
